src/api/json/catalog.json

```diff
@@ -1109,13 +1109,18 @@
       "url": "https://raw.githubusercontent.com/iterative/dvcyaml-schema/master/schema.json"
     },
     {
-      "name": "Eclipse Che Devfile",
-      "description": "JSON schema for Eclipse Che Devfiles",
-      "url": "https://raw.githubusercontent.com/eclipse-che/che-server/master/wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json",
+      "name": "Devfile",
+      "description": "JSON schema for Devfiles",
+      "url": "https://raw.githubusercontent.com/devfile/api/2.1.x/schemas/latest/devfile.json",
       "fileMatch": [
         "devfile.yaml",
         ".devfile.yaml"
-      ]
+      ],
+      "versions": {
+        "1.0.0": "https://raw.githubusercontent.com/eclipse-che/che-server/master/wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json",
+        "2.0.0": "https://raw.githubusercontent.com/devfile/api/2.0.x/schemas/latest/devfile.json",
```

Comment:
could you clarify why branches are used, and not tags e.g. `v2.0.0` / `v2.1.0`  ?

Reply:
I've updated the links from branches to tags, it should now be on v2.0.0 and not 2.0.x!